packages/core/interfaces/src/index.ts

```diff
@@ -158,6 +165,60 @@ export interface UseLocale
   loading: ComputedProperty<boolean>;
 }
 
+export interface ProductGetters<PRODUCT, PRODUCT_FILTER> {
+  getName: (product: PRODUCT | Readonly<PRODUCT>) => ComputedProperty<string>;
+  getSlug: (product: PRODUCT | Readonly<PRODUCT>) => ComputedProperty<string>;
+  getPrice: (product: PRODUCT | Readonly<PRODUCT>) => ComputedProperty<number | null>;
```

Comment:
Unless we have to deal with regular price only, we need to return a object of prices (regular: 100$, finalPrice: 80%), or add some other getter for special price - after discount, for example.

Reply:
this is important!